src/sentry/utils/circuit_breaker2.py

```diff
@@ -29,7 +29,7 @@
 DEFAULT_RECOVERY_WINDOW_MULTIPLIER = 2
 
 
-class CircuitBreakerState(Enum):
+class CircuitBreakerState(str, Enum):
```

Comment:
could we not use `state.value` to get the string value of the enum?

Reply:
Good call. Fixed it in the follow-up PR, and removed the type change here.